build/gen_stub.php

```diff
@@ -1277,7 +1281,9 @@ private function getFlagsAsString(): string
             $flags[] = "ZEND_ACC_FINAL";
         }
 
-        if ($this->flags & Class_::MODIFIER_ABSTRACT) {
+        if ($this->isExplicitAbstract) {
+            $flags[] = "ZEND_ACC_EXPLICIT_ABSTRACT_CLASS";
```

Comment:
EXPLICIT_ABSTRACT_CLASS and ABSTRACT are the same, so it doesn't make sense to track this separately. You can always use ZEND_ACC_ABSTRACT.

Reply:
That's cool! At the first sight, it looked like there is a distinction.